src/PageGenerator/Program.cs

```diff
@@ -35,20 +35,31 @@ public void Main(string[] args)
                     "Unable to open library {0}. Is it spelled correctly and listed as a dependency in project.json?",
                     args[0]));
             }
-            var viewBasePath = Path.Combine(Path.GetDirectoryName(diagnosticsLibInfo.Path), "Views");
 
-            Console.WriteLine("Generating code files for views in {0}", viewBasePath);
-            Console.WriteLine();
-
-            var cshtmlFiles = GetCshtmlFiles(viewBasePath);
+            var viewDirectories = Directory.EnumerateDirectories(
+                Path.GetDirectoryName(diagnosticsLibInfo.Path), "Views", SearchOption.AllDirectories);
 
             var fileCount = 0;
-            foreach (var fileName in cshtmlFiles)
+            foreach (var viewDir in viewDirectories)
             {
-                Console.WriteLine("  Generating code file for view {0}...", Path.GetFileName(fileName));
-                GenerateCodeFile(fileName, string.Format("{0}.Views", args[0]));
-                Console.WriteLine("      Done!");
-                fileCount++;
+                Console.WriteLine();
```

Comment:
These changes were required as I rearranged the (*.cshtml) files in the project to be present in their own respective folders. After the changes:

![image](https://cloud.githubusercontent.com/assets/6559784/9666256/d6724622-5229-11e5-891f-9a76755b7fdb.png)